Assets/Mirage/Runtime/NetworkSceneManager.cs

```diff
@@ -19,161 +19,152 @@ namespace Mirage
     [DisallowMultipleComponent]
     public class NetworkSceneManager : MonoBehaviour, INetworkSceneManager
     {
+        #region Fields
+
         static readonly ILogger logger = LogFactory.GetLogger(typeof(NetworkSceneManager));
 
+        [Header("Setup Settings")]
+
         [FormerlySerializedAs("client")]
         public NetworkClient Client;
+
         [FormerlySerializedAs("server")]
         public NetworkServer Server;
 
         /// <summary>
-        /// Sets the NetworksSceneManagers GameObject to DontDestroyOnLoad. Default = true.
+        ///     Sets the NetworksSceneManagers GameObject to DontDestroyOnLoad. Default = true.
         /// </summary>
         public bool DontDestroy = true;
 
+        public readonly Dictionary<Scene, List<INetworkPlayer>> ServerSceneData = new Dictionary<Scene, List<INetworkPlayer>>();
```

Comment:
does this collection need to be public? would be better if there were get only methods for stuff if users need to. so that they can't modify the collection.

Reply:
Good call I can make a method to get the info they need.